lib/documents/schemas/asylum_support_decisions.json

```diff
@@ -111,6 +111,10 @@
           "value": "section-4-2-jurisdiction"
         },
         {
+          "label": "Section 4(2) - other",
+          "value": "section-4-2-other"
+        },
+        {
           "label": "Section 4(2) - regulation 3(2) - other",
```

Comment:
To be clear, "label": "Section 4(2) - regulation 3(2) - other" should not exist.
Only "label": "Section 4(2) - other" should exist